AWSAuthSDK/Sources/AWSAuthUI/AWSAuthUIViewController.m

```diff
@@ -128,11 +128,7 @@ + (AWSAuthUIConfiguration *)getDefaultAuthUIConfiguration {
 
     if ([AWSAuthUIViewController isConfigurationKeyPresent:AWSInfoAppleIdentifier]) {
         Class appleClass = NSClassFromString(@"AWSAppleSignInButton");
-        if (appleClass) {
-            [authUIConfig addAWSSignInButtonViewClass:appleClass];
-        } else {
-            AWSDDLogWarn(@"Found Apple sign in configuration in awsconfiguration.json but could not find dependencies. Skipping rendering in AuthUI");
-        }
+        [authUIConfig addAWSSignInButtonViewClass:appleClass];
```

Comment:
I think you can do away with the `NSClassFromString` lookup too, right? `[AWSAppleSignInButton class]` should return the class object you need.